quinn-h3/src/proto/headers.rs

```diff
@@ -197,10 +219,10 @@ pseudo_type![
 #[derive(Debug, Default, Eq, PartialEq)]
 pub struct Pseudo {
     // Request
-    pub method: Option<Method>,
-    pub scheme: Option<String<Bytes>>,
-    pub authority: Option<String<Bytes>>,
-    pub path: Option<String<Bytes>>,
+    pub(self) method: Option<Method>,
```

Comment:
Not sure what `pub(self)` is or why it's needed here?

Reply:
Wasn't needed, thanks.